src/lib/esnext.intl.d.ts

```diff
@@ -1,3 +1,7 @@
 declare namespace Intl {
-   // Empty for now
+
+    interface NumberFormat {
+        formatRange(startDate: number | bigint, endDate: number | bigint): string;
+        formatRangeToParts(startDate: number | bigint, endDate: number | bigint): NumberFormatPart[];

```

Comment:
From reading the spec, I see that [the parameter names are `start` and `end`](https://tc39.es/proposal-intl-numberformat-v3/out/numberformat/proposed.html#sec-intl.numberformat.prototype.formatrange).

```suggestion
        formatRange(start: number | bigint, end: number | bigint): string;
        formatRangeToParts(start: number | bigint, end: number | bigint): NumberFormatPart[];
```